### DISCUSSION TITLE
Document field update

### DISCUSSION THREAD

**Discussion by kepatopoc** [quickwit-oss/quickwit#3421] [Q&A] [answered]
Is it possible now in quickwit to update a field on a document? [Like in Elastic](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update.html)

`{'doc_id':1, 'text':'it is not possible to update a document field in this database','tag_name':[]}`
after UPDATE add tag 'negative' and 'feature'
`{'doc_id':1, 'text':'...', 'tag_name':['negative','feature']}`

If not, is it planned in the future?

**Comment by evanxg852000** [comment, marked as answer]
Hi @kepatopoc, sorry for the late reply.
Unfortunately, we have no plan on adding this feature due. I believe this is related to the way [Tantivy](https://github.com/quickwit-oss/tantivy) the search engine library powering Quickwit works.
This feature could become crucial; but for now, we have no plan for it.

**Reply by kepatopoc** [reply]
Thanks for the answer. I hope there will be such an opportunity in the future. Good luck in development!
